src/styles/globals.css

```diff
@@ -0,0 +1,9 @@
+:root{
```

Comment:
These colours look great! I like my choices but do feel we need another colour that would look good with them?